config/kubernetes-csi/kustomization.yaml

```diff
@@ -3,3 +3,7 @@ kind: Kustomization
 bases:
   - ../kubernetes
   - ../common/csi
+images:
```

Comment:
wouldn't it be easier to move this patch into `common/csi` and only have it once?

Reply:
it would be yes, you are right